recipes/cctag/all/conanfile.py

```diff
@@ -59,6 +59,7 @@ def layout(self):
         cmake_layout(self, src_folder="src")
 
     def requirements(self):
+        # boost/1.85.0 not compatible because of "error: 'numeric' is not a namespace-name" error
         self.requires("boost/1.84.0", transitive_headers=True, transitive_libs=True)
```

Comment:
```suggestion
        if Version(self.version) >= "1.0.4":
            self.requires("boost/1.84.0", transitive_headers=True, transitive_libs=True)
        else:
            # boost/1.85.0 not compatible because of "error: 'numeric' is not a namespace-name" error
            self.requires("boost/1.84.0", transitive_headers=True, transitive_libs=True)
```

It seems like the new patch version fixed that problem: https://github.com/alicevision/CCTag/compare/v1.0.3...v1.0.4#diff-d975bf659606195d2165918f93e1cf680ef68ea3c9cab994f033705fea8238b2R30

Reply:
Great catch!